app/services/data/get-offender-manager-utilisation.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function (userId, year) {
```

Comment:
Just extending the comment above - if we had this be `(orgunittype, id, year)` we can start with offender manager and extend to include the others later.